src/runtime/cli/config/configuration-qemu.toml.in

```diff
@@ -309,7 +309,7 @@ pflashes = []
 # lexicographical order, to the lifecycle of the guest container.
 # Hooks are executed in the runtime namespace of the guest. See the official documentation:
 # https://github.com/opencontainers/runtime-spec/blob/v1.0.1/config.md#posix-platform-hooks
-# Warnings will be logged if any error is encountered will scanning for hooks,
+# Warnings will be logged if any error is encountered while scanning for hooks,
```

Comment:
Eric, could you also do the same fix for the https://github.com/kata-containers/kata-containers/blob/14bb24e4ca4d097c6374901760a50183026682ab/src/runtime/cli/config/configuration-acrn.toml.in#L116 ?